### DISCUSSION TITLE
I lost the 2FA recovery code for my account with a paid subscription.

### DISCUSSION THREAD

**Discussion by bespinga** [community/community#144390] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The administrator account of the organization with an active paid subscription (Copilot Business) is unable to log in due to a lost 2FA recovery code. Automatic monthly charges are being incurred, but user management is not possible in this state.

We've been reaching out to GitHub Support for two months, but have yet to receive a response. Is submitting a support request to GitHub Support the only way to resolve this account issue?

유료 구독 (Copilot Business) 카드 자동 과금이 되고 있는 Organization 관리자 계정의 2FA Recovery Code를 분실하여 로그인을 못하고 있습니다. 사용자 관리를 못하고 있는 상태에서 매월 카드 자동결제가 발생하고 있습니다.

Github Support에 2달째 문의를 등록하고 있는데, 답변이 없습니다.
계정문제를 해결하려면 Github Support에 문의 등록 하는 방법밖에 없나요?

**Comment by Rawknee-69** [comment]
well ig here is the step by step guide you should follow and if you found this helpful please mark this as an answer @bespinga 

GitHub typically provides several options to verify your identity if you've lost your 2FA recovery codes. Here are some things to try:

Authenticator App Backup Codes: If you set up backup codes when enabling 2FA, you can use one of those codes to log in.
SMS 2FA: If you’ve set up SMS-based two-factor authentication, you might still be able to log in using the phone number linked to your account.
Recovery using the Registered Email: GitHub offers a recovery flow via email, where they can help verify your identity and disable 2FA temporarily.
Contact GitHub Support (Again):
Since you've already reached out and haven’t received a response, it's frustrating, but sometimes support queues can take a while. You could try submitting a new request, referencing your previous ticket. Be clear and detailed about the urgency of the situation—mention that you're unable to manage user accounts and there are ongoing charges.

You can reach GitHub Support directly at:

[GitHub Support Page](https://support.github.com/contact)
Check GitHub’s Help Documentation:
GitHub has a [2FA recovery guide](https://docs.github.com/en/github/authenticating-to-github/recovering-your-account-from-lost-2fa) that might help in the case of lost 2FA codes. This can sometimes be the fastest route if you’re unable to reach support right away.

Consider a Subscription Charge Dispute:
If you are unable to access the admin account to stop the monthly charges, and if it’s been an extended period, you might want to reach out to GitHub's billing team separately to dispute the charges or pause the subscription while you regain access to the account.

Patience & Persistence:
If all else fails, continue following up with GitHub Support, but also be patient as they work through the backlog.
